File: cfgs/java_cfgs_foo/NUMBER_SUBSEQUENCES_FORM_AI_BJ_CK.java
Nodes
1;special;;;
2;statement;int aCount = 0;14;
3;statement;int bCount = 0;15;
4;statement;int cCount = 0;16;
5;statement;int i = 0;17;
6;conditional;foo_1 < fun_1();18;
7;conditional;fun_1(foo_2) == 'a';20;
8;statement;aCount = 1 + (2 * foo_1);20;
9;conditional;fun_1(foo_2) == 'b';21;
10;statement;bCount = foo_1 + (2 * foo_2);21;
11;conditional;fun_1(foo_2) == 'c';22;
12;statement;cCount = foo_1 + (2 * foo_2);22;
13;statement;i++;19;
14;statement;return foo_1;24;
15;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;
5;7;14;
6;8;9;
7;13;
8;10;11;
9;13;
10;12;13;
11;13;
12;6;
13;15;
14;

File: cfgs/pythoncfgs/NUMBER_SUBSEQUENCES_FORM_AI_BJ_CK.py
Nodes
1;statement;foo_1 = 0;8;
2;statement;foo_1 = 0;9;
3;statement;foo_1 = 0;10;
4;conditional;fun_2(fun_2(foo_1));11;
5;statement;return foo_1;18;
6;conditional;foo_1[foo_2] == 'a';12;
7;conditional;foo_1[foo_2] == 'b';14;
8;conditional;foo_1[foo_2] == 'c';16;
9;statement;foo_1 = foo_2 + 2 * foo_3;17;
10;statement;foo_1 = foo_2 + 2 * foo_3;15;
11;statement;foo_1 = 1 + 2 * foo_2;13;
12;exit;;
Edges
0;2;
1;3;
2;4;
3;5;6;
4;12;
5;7;11;
6;8;10;
7;4;9;
8;4;
9;4;
10;4;
11;
